2_accept_offer.py

```diff
@@ -93,7 +93,11 @@ def process_offer(offer, audit):
     + locale.format("%.8f", Decimal(offer_currency_quantity) / COIN, True) + " " + offer_currency_code + " for " \
     + locale.format("%.8f", Decimal(ask_currency_quantity) / COIN, True) + " " + ask_currency_code
 
-  # TODO: Prompt the user for whether the trade is acceptable
+  answer = raw_input("Do you want to continue with the trade? 'Y' to continue, 'N' to quit.  ").lower()
+  while answer not in ['y', 'yes', 'n', 'no']:
+    answer = raw_input("Not recognised. 'Y' to continue, 'N' to quit.  ").lower()
+  if answer != 'y' or answer != 'yes':
```

Comment:
Also I am not sure why I even tested against yes/y in the first place rather than a far more sensible no/n!

Sorry :( I'm usually better than this so maybe you don't want me near your stuff!

I also left the hot water tap running all day yesterday and nearly flooded the kitchen so it it follows a trend right now.

And I can't sleep lol!


> On 7 Jan 2015, at 22:54, Ross Nicoll <notifications@github.com> wrote:
> 
> In 2_accept_offer.py:
> 
> > @@ -93,7 +93,11 @@ def process_offer(offer, audit):
> >      + locale.format("%.8f", Decimal(offer_currency_quantity) / COIN, True) + " " + offer_currency_code + " for " \
> >      + locale.format("%.8f", Decimal(ask_currency_quantity) / COIN, True) + " " + ask_currency_code
> >  
> > -  # TODO: Prompt the user for whether the trade is acceptable
> > +  answer = raw_input("Do you want to continue with the trade? 'Y' to continue, 'N' to quit.  ").lower()
> > +  while answer not in ['y', 'yes', 'n', 'no']:
> > +    answer = raw_input("Not recognised. 'Y' to continue, 'N' to quit.  ").lower()
> > +  if answer != 'y' or answer != 'yes':
> Shouldn't this be "and"? As is, it wants it to be both 'y' and 'yes' in order to not exit.
> 
> \
> Reply to this email directly or view it on GitHub.
>